.github/PULL_REQUEST_TEMPLATE/pull_request_template.md

```diff
@@ -0,0 +1,16 @@
+---
+name: Pull request
+about: Create a pull request to contribute new features or bug fixes
+title: "[PULL REQUEST]"
+labels: pull_request
+
+---
+
+## Describe your changes
```

Comment:
Should we have a section for describing the intent of the PR? This section is the "what", but what about the "why"?

Reply:
I feel it is too much to ask every PR to explicitly include a section that describes why. I can consolidate this to say "Describe your changes and why you are making these changes"